app/apphandlers/setup.go

```diff
@@ -178,9 +180,10 @@ func monitorDestRouters(ctx context.Context, routerFactory *router.Factory, batc
 	routerFactory.RouterDB.DeleteExecuting()
 	batchRouterFactory.RouterDB.DeleteExecuting()
 
-	for config := range ch {
-		sources := config.Data.(backendconfig.ConfigT)
+	for workspaceConfig := range ch {
```

Comment:
**Question:** why do we need both `apphandlers.monitorDestRouters` and `LifecycleManager.monitorDestRouters`?